.github/container/Dockerfile.t5x.arm64

```diff
@@ -0,0 +1,226 @@
+# syntax=docker/dockerfile:1-labs
+# Example command to build manually:
+#   docker buildx build -f Dockerfile.t5x.arm64 --tag t5x --build-arg BASE_IMAGE=ghcr.io/nvidia/jax:mealkit-2023-12-12 .
+
+ARG BASE_IMAGE=ghcr.io/nvidia/jax:mealkit
+ARG REPO_T5X=https://github.com/google-research/t5x.git
+ARG REF_T5X=main
+ARG SRC_PATH_T5X=/opt/paxml
+
+###############################################################################
+## build array_record and grain which do not have working arm64 pip wheels
+###############################################################################
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as wheel-builder
+
+# grain build needs bazel
+RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-arm64 -O /usr/bin/bazel && \
+    chmod a+x /usr/bin/bazel
+
+#------------------------------------------------------------------------------
+# build array_record 0.5.0 from source
+#------------------------------------------------------------------------------
+# TODO: Remove this once array_record maintainers have published an arm64 wheel.
+
+# The bazel build of array_record makes some strong assumptions on where the cpp toolchain lives.
+# We therefore stick to its existing Dockerfile, which ensures everything is in the right place:
+#
+#   https://github.com/google/array_record/tree/v0.5.0/oss/build.Dockerfile.aarch64
+#
+FROM linaro/tensorflow-arm64-build:2.12-multipython as array_record-builder
+# TODO: Infer PYTHON_VERSION from our own $BASE_IMAGE?
```

Comment:
Yes please 😄

Reply:
A possible solution is to write the Python version number to a file in the wheel-builder stage, and then use `COPY --from=wheel-builder ...` to copy this information to the new stage.